api/lib/mno_enterprise/concerns/controllers/jpi/v1/admin/impac/tenant_dashboards_controller.rb

```diff
@@ -0,0 +1,91 @@
+module MnoEnterprise::Concerns::Controllers::Jpi::V1::Admin::Impac::TenantDashboardsController
+  extend ActiveSupport::Concern
```

Comment:
lots of code duplicated betweend tenant_dashboard_controller and dashboard_controller. I think it would be worth having a DashboardControllerHelper in order to put in common all the methods. only the dashboards and dashboard_params would be defined in the controller.